src/ToDo.UI/Views/Dialogs/ExpirationDateFlyout.xaml

```diff
@@ -33,114 +102,99 @@
 		<utu:AutoLayout Padding="0,8,0,8"
 						CornerRadius="4">
 			<utu:AutoLayout Orientation="Horizontal">
-				<utu:AutoLayout Padding="8,8,8,8"
-								Justify="SpaceBetween"
-								Orientation="Horizontal"
-								utu:AutoLayout.CounterAlignment="Start"
-								utu:AutoLayout.PrimaryAlignment="Stretch">
-					<utu:AutoLayout Spacing="10"
-									Padding="0,0,16,0"
-									Orientation="Horizontal"
-									utu:AutoLayout.CounterAlignment="Center">
-						<PathIcon Data="{StaticResource Icon_Calendar_Today}"
-								  utu:AutoLayout.CounterAlignment="Start" />
-					</utu:AutoLayout>
-					<utu:AutoLayout Spacing="10"
-									Padding="0,2,0,2"
+				<Button Style="{Binding MaterialTextButtonStyle}" BorderThickness="0" Command="{Binding Today}">
+					<utu:AutoLayout Padding="8,8,8,8"
+									Justify="SpaceBetween"
 									Orientation="Horizontal"
-									utu:AutoLayout.CounterAlignment="Center"
+									utu:AutoLayout.CounterAlignment="Start"
 									utu:AutoLayout.PrimaryAlignment="Stretch">
-						<TextBlock Foreground="{ThemeResource OnSurfaceBrush}"
-								   Text="Today"
-								   utu:AutoLayout.CounterAlignment="Center"
-								   Style="{StaticResource BodyMedium}"
-								   x:Uid="Task_Details_Add_Due_Date_Label" />
+						<utu:AutoLayout Spacing="10"
+										Padding="0,0,16,0"
+										Orientation="Horizontal"
+										utu:AutoLayout.CounterAlignment="Center">
+							<PathIcon Data="{StaticResource Icon_Calendar_Today}"
+									  utu:AutoLayout.CounterAlignment="Start"/>
+						</utu:AutoLayout>
+						<utu:AutoLayout Spacing="10"
+										Padding="0,2,0,2"
+										Orientation="Horizontal"
+										utu:AutoLayout.CounterAlignment="Center"
+										utu:AutoLayout.PrimaryAlignment="Stretch">
+							<TextBlock
+									   Text="Today"
+								FontWeight="Bold"
+									   utu:AutoLayout.CounterAlignment="Center"
+									   x:Uid="Task_Details_Add_Due_Date_Label" />
```

Comment:
1. So far the naming convention for the x:Uid is NameOfPage_DescriptionOfElement.ElementProperty
2. You should update the resources.l12n.xml and resources.resw files to add the localization.

Reply:
tooltip